terminus-ui/scss/helpers/_color.spec.scss

```diff
@@ -9,80 +9,65 @@
     @include assert-equal($test, $expect);
   }
 
-  /*   @include test('should return the correct theme (this seems to fail due to class being added by true)') {
-    @include assert{
+  @include test('should return the correct theme and color for background-color') {
+    @include assert {
       @include output {
         @include theme-color(background-color);
       }
       @include expect {
```

Comment:
Any reason we seem to be testing Sass code here but testing pure CSS output in our other tests?

Reply:
The classes added by true are not something we should be explicitly looking for, and when taking straight output, I was getting errors with true-added-class `.test-output`

In general, I'm not concerned about testing mixins or functions within mixins, but this one is native sass, so I will go back, since I know more now, to see if there's a better way of testing this.